VideoAPI/VideoApi.UnitTests/Controllers/Participant/GetDistinctJudgeNamesTests.cs

```diff
@@ -0,0 +1,41 @@
+
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using VideoApi.Contract.Responses;
+using VideoApi.DAL.Queries;
+
+namespace VideoApi.UnitTests.Controllers.Participant
+{
+    public class GetDistinctJudgeNamesTests : ParticipantsControllerTestBase
+    {
+
+        [SetUp]
+        public void TestInitialize()
+        {
+            var judgeFirstnames = new List<string> { "JudgeName1", "JudgeName2", "JudgeName3" };
+
+            MockQueryHandler
+                .Setup(x => x.Handle<GetDistinctJudgeListByFirstNameQuery, List<string>>(It.IsAny<GetDistinctJudgeListByFirstNameQuery>()))
+                .ReturnsAsync(judgeFirstnames);
+        }
+
+        [Test]
+        public async Task should_getjudgelistResponses()
```

Comment:
are you using the linting? Methods should start with a capital letter and you have a mix of camelcase and underscores

Reply:
Should_get_judge_list_response or ShouldGetJudgeListResponse